### ISSUE TITLE
Product Page Feedback: Accessibility error

### ISSUE THREAD

**Issue by umarmughal824** [mitodl/bootcamp-ecommerce#698]
- Accessibility: there should be only one H1 per page. "How to Apply" is a H1 now and probably should be an H2.
